.nowignore

```diff
@@ -0,0 +1,30 @@
+*
```

Comment:
This file looks a little intense at first glance. I'm sure it's necessary but it's not clear to me why without having any familiarity with Now. 

Are there any circumstances going forward where we'd need to make updates here (and if so what are those so I can try to commit them to memory 😉), or is this pretty much set for the forseeable future?

Reply:
Now ignores seem to be totally broken; I'm deferring it for now: https://github.com/badges/shields/pull/2906#issuecomment-460457391

Reply:
Oops! I didn't put two and two together there